pages/my/daos/MyDaosPage.module.scss

```diff
@@ -25,15 +25,14 @@
   @include tablet-portrait {
     align-items: center;
     flex-direction: row;
-    gap: 35px;
   }
 
   h1 {
     font-family: var(--font-family-tertiary);
-    margin: 0 24px 0 0;
```

Comment:
I consider we should check styles/layouts of both pages and move them to shared component if possible or just move styles to sharable file.